frontend/src/lib/candidate/components/logoutButton/LogoutButton.svelte

```diff
@@ -78,7 +82,11 @@ When set to false, the button variant is ghost.
     text={$t('candidateApp.navbar.logOut')}
     color="warning" />
 {:else}
-  <Button on:click={triggerLogout} text={$t('candidateApp.homePage.logOut')} color="warning" />
+  <Button
+    on:click={triggerLogout}
+    text={$t('candidateApp.homePage.logOut')}
+    color="warning"
+    variant="main" />
```

Comment:
This component is also used, for example, in the front page, which now looks like this:
![image](https://github.com/OpenVAA/voting-advice-application/assets/122277053/14f84c6f-b3fa-4a8d-9ae5-d6d5cc887743)

In the figma designs the style of the front page log out button is different so setting the variant to "main" here should probably be done in a different way. One fix would be that the variant could be given as props to the LogoutButton.